hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -68,11 +68,16 @@
   public static final String HDDS_CONTAINERSCRUB_ENABLED =
       "hdds.containerscrub.enabled";
   public static final boolean HDDS_CONTAINERSCRUB_ENABLED_DEFAULT = false;
+
   public static final boolean HDDS_SCM_SAFEMODE_ENABLED_DEFAULT = true;
   public static final String HDDS_SCM_SAFEMODE_MIN_DATANODE =
       "hdds.scm.safemode.min.datanode";
   public static final int HDDS_SCM_SAFEMODE_MIN_DATANODE_DEFAULT = 1;
 
+  public static final String HDDS_CONTAINER_SCANNER_VOLUME_BYTES_PER_SECOND =
+      "hdds.container.scanner.volume.bytes.per.second";
+  public static final long
+      HDDS_CONTAINER_SCANNER_VOLUME_BYTES_PER_SECOND_DEFAULT = 1048576L;
 
```

Comment:
1048576L , ozone supports writing things like 1MB. if that is useful.

Reply:
ok. let me use that.